stl.go

```diff
@@ -362,15 +375,15 @@ func parseGSIBlock(b []byte) (g *gsiBlock, err error) {
 	if v := strings.TrimSpace(string(b[224:230])); len(v) > 0 {
 		if g.creationDate, err = time.Parse("060102", v); err != nil {
 			err = fmt.Errorf("astisub: parsing date %s failed: %w", v, err)
-			return
+			g.creationDate = time.Now()
```

Comment:
I don't like ignoring this error. Which value of `b[224:230]` is giving trouble ?

Reply:
We have some olf stl files in Archive that does not have this parameter filled and for us it is more important to convert into webvtt than knowing the date